src/client/mod.rs

```diff
@@ -1,23 +1,11 @@
-use std::{
-    collections::BTreeMap,
-    convert::Infallible,
-    sync::Arc,
-};
+use std::{collections::BTreeMap, convert::Infallible, sync::Arc};
```

Comment:
there's a rustfmt.toml in the root of the repo here that should make these changes go away if rustfmt'd with it.

Is it possible for you to just run rustfmt once so it's easier to review?